src/api/fetch.js

```diff
@@ -4,26 +4,29 @@ const fetchRetry = require("fetch-retry")(fetch);
 export async function* indexGenerator(path, auth) {
   let page = 1;
   while (true) {
-    const request = await fetchRetry(`${baseURL}/${path}?page=${page}`, {
-      retries: 5,
-      retryDelay: function (attempt) {
-        return Math.pow(2, attempt) * 15000;
-      },
-      method: "GET",
-      headers: {
-        "x-secret": auth.secret,
-        "x-device-id": auth.device_id,
-      },
-    }).catch((reason) => {
+    let response;
```

Comment:
The same should be done here, but GitHub won't let me edit a section this large.

Reply:
same as above, this doesn't work